hasher-matcher-actioner/terraform/collabs_example.json

```diff
@@ -9,6 +9,9 @@
             },
             "tags" : {
                 "L" : []
+            },
+            "active" : {
```

Comment:
Can we specify what `active` means? Maybe `fetcher_active` or `keep_in_sync`

Reply:
yeah I think fetcher_active makes sense